servers/bitwindow/main.go

```diff
@@ -128,6 +140,9 @@ func realMain(ctx context.Context) error {
 		return err
 	}
 
+	bitcoinEngine := bitcoind_engine.New(proxy, db)
+	go bitcoinEngine.Run(ctx)
```

Comment:
I really think that this run method should return an error (i.e. errors we cannot recover from), and that those error should be drained into `errs`